theme/styles/global.scss

```diff
@@ -25,15 +25,20 @@
 //* Product Grid */
 .grid {
   display: grid;
-    padding-left: 0.125rem;
-    padding-right: 0.125rem;
-    gap: 0.125rem;
-    grid-template-columns: 1fr;
+  padding-left: 0.125rem;
+  padding-right: 0.125rem;
+  gap: 0.125rem;
+  grid-template-columns: 1fr;
 
-    @include media('md-up') {
-      grid-template-columns: repeat(3, 1fr);
-    }
+  @include media('md-up') {
+    grid-template-columns: repeat(3, 1fr);
+  }
 
+  &--style-related-products {
```

Comment:
Up to you, but I think it's better to have these modifiers describe what they are modifying, versus what module they apply to. 

For example, if there ends up being another grid-style module, that omits the padding-left/padding-right, the `--style-related-products` modifier is no longer semantically correct, since the new module is not a `related-products` module, but something else. 

Instead, something like `.grid--padding-x-0` and `.grid--first-cell-enlarged` are both more descriptive when reading the class name, and can apply to any module type. 

Just a thought!

Reply:
THANKS! This is super helpful because I have actually been wondering about this for awhile now and always end up naming them something generic or after modules/specific variants of modules that prob don't make a lot of sense!!